tiatoolbox/visualization/tileserver.py

```diff
@@ -37,7 +39,7 @@ class TileServer(Flask):
         >>> app.run()
     """
 
-    def __init__(self, title: str, layers: Dict[str, WSIReader]) -> None:
+    def __init__(self, title: str, layers: Union[Dict, List]) -> None:
```

Comment:
>I did this so that it isn't necessary to give names to the layers. You can give it a list and have autogenerated layer names ('layer1', 'layer2', etc), or a dict which specifies names for each layer via the key.

Although, thinking on that, there isn't a provision for specifying the layer names in the cli command, and there probably should be. I can add that in later today.

Will change the hint as suggested also.